### PR TITLE
vmui: change the response for active queries

### PR DESCRIPTION
**Pull request by Loori-R**
Change the response to valid json for active queries

Fixed this bug: https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4676#issuecomment-1663203424

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]

File: app/vmselect/promql/active_queries.go [hunk 1]
```diff
@@ -23,8 +24,12 @@ func ActiveQueriesHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, `{"status":"ok","data":[`)
 	for i, aqe := range aqes {
 		d := now.Sub(aqe.startTime)
+		addr := aqe.quotedRemoteAddr
+		if n := strings.IndexByte(aqe.quotedRemoteAddr, ','); n != -1 {
```
Comment (hagen1778) [inline app/vmselect/promql/active_queries.go:28]: I don't understand how this change is related to the mentioned [comment](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4676#issuecomment-1663203424). 
Would you mind adding some explanations or tests? Although, would using of `%q` formatting on L:31 help here?

**Review by hagen1778 — COMMENTED** [review]
Would be nice to add a note to changelog

File: app/vmselect/promql/active_queries.go (hunk 1 above)
Comment (Loori-R) [reply app/vmselect/promql/active_queries.go:28]: `aqe.quotedRemoteAddr` can be `value` or `value, X-Forwarded-For: value`. 
If you use `%q` formatting, the value will be `remote_addr: "value, X-Forwarded-For: value"`

https://github.com/VictoriaMetrics/VictoriaMetrics/blob/bd8ecfb5518936687c8f24cbc82a72069e50ea6b/lib/httpserver/httpserver.go#L437-L444

**Review by hagen1778 — APPROVED** [review]
LGTM!
Would you mind adding a changelog note?

**Comment by Loori-R** [comment]
@hagen1778 A note has been added to the changelog.

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Comment by valyala** [comment]
FYI, the fix has been reverted in the commit ac0b7e042152572f47e72d70de5293ed81149f8b , because it wasn't correct. The correct fix has been included in be5c4818f51a9eb426dae121cd3dc13218be947b .

**Pull request merged**